File: solutions/0856-consecutive-numbers-sum/solution.py
class Solution(object):
    def consecutiveNumbersSum(self, n):
        """
        :type n: int
        :rtype: int
        """
        count,i=0,1;
        while(n>0):
            n=n-i;
            if(n%i==0):
                count+=1;
            i+=1;
        return count;

File: solutions/0856-consecutive-numbers-sum/solution.js
/**
 * @param {number} n
 * @return {number}
 */
var consecutiveNumbersSum = function(n) {
    let count=0,i=1;
    while(n>0)
    {
        n=n-i;
        if(n%i==0)
            count++;
        i++;
    }
    return count;
};
